src/main/java/frc/robot/RobotContainer.java

```diff
@@ -112,16 +114,22 @@ public void putShuffleboardCommands() {
 
         }
         if (Constants.FEEDER_DEBUG){
-            SmartShuffleboard.putCommand("Feeder", "Feed", new StartFeeder(feeder));
+//            SmartShuffleboard.putCommand("Feeder", "Feed", new StartFeeder(feeder));
```

Comment:
?

Reply:
feeder causes issues on the 2023 robot. code immedatly crashes.

Reply:
Is Main broken? This should be fixed - not disabled. You should be able to test locally like that but not commit back.